### ISSUE TITLE
Unlocked lockboxed become locked when mailed 

### ISSUE THREAD

**Issue by nemmjuuk** [dalaranwow/dalaran-wow#4688]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Unlock a lockbox or junkbox. Mail it to someone else. The unlocked box has magically become locked again.

**Expected behaviour**: Tell us what should happen instead
Mailing shouldn't alter the state of the box. 
**Steps to reproduce the problem**:
See above.

**Include proofs for this behaviour**
Googled this issue, but couldn't find any indication that mailing lockboxes has ever been a problem in retail. 

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Confirmed.

**Comment by nemmjuuk** [comment]
Btw: this has implications when it comes to leveling lockpicking skills.

**Comment by DrFuler** [comment]
**Update:** Just checked and it STILL happens with [Heavy Junkbox](http://www.wowhead.com/item=16885/heavy-junkbox), but it DOESN'T HAPPEN with [Strong Junkbox](http://www.wowhead.com/item=29569/strong-junkbox) !

When [Strong Junkbox](http://www.wowhead.com/item=29569/strong-junkbox) is in the mail, it always looks locked. But when you move it to your personal bags it returns to its normal state.

_Printscreen 1:_ https://ibb.co/eTjXRn
_Printscreen 2:_ https://ibb.co/dHkHsS
